students/johnpharmd/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,13 @@
+def fizzbuzz():
+    n = 1
+    while n < 101:
+        if n % 3 == 0 and n % 5 == 0:
+            print('FizzBuzz')
+        elif n % 3 == 0:
+            print('Fizz')
+        elif n % 5 == 0:
+            print('Buzz')
+        else:
+            print(n)
+        n += 1
+
```

Comment:
Looks good! Just for fun, can you think of a way to write this where you only evaluate for %3 and %5 once?